tlatools/org.lamport.tlatools/customBuild.xml

```diff
@@ -163,6 +199,11 @@
 			release="${java.release}"
 			encoding="UTF-8"
 		>
+			<!-- Exclude the files compiled separately up above to silence warnings. -->
+			<exclude name="tlc2/tool/fp/LongArray.java" />
+			<exclude name="tla2sany/parser/SimpleCharStream.java" />
```

Comment:
`SimpleCharStream` appears to be the source of [newly introduced warnings](https://github.com/tlaplus/tlaplus/commit/0f27f0bdb4864ec92654abf49362e565e332b71e).  Why can they be ignored?

Reply:
It's a JavaCC-generated file so the only way to fix the warnings is to upgrade the version of JavaCC to one that doesn't generate code with warnings